CHANGELOG.md

```diff
@@ -13,6 +13,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - 
 - 
 - 
+- Add warning on trying to pass numeric timestamp if freq is not None and add `_cast_index_to_datetime` ([#214](https://github.com/etna-team/etna/pull/214))
+- 
```

Comment:
May be we should edit somehow CHANGELOG.md? Now, there are a lot of blank lines.

Reply:
We remove blank lines after release usually. I don't think blank lines are a problem.